static/scripts/autocomplete.js

```diff
@@ -0,0 +1,67 @@
+function autocomplete(datalist, worker) {
+    var processing = false;
+    var nextQuery = null;
+    var currentQuery = null;
+
+    var prof_query = document.getElementById('prof_query');
+    var suggestions = document.getElementById('suggestions');
+
+
+    // fuzzy search
+    worker.postMessage({type: 'data', data: datalist});
+
+    worker.onmessage = function(results) {
+        displayResult(results.data);
+
+        processing = false;
+        if (nextQuery !== null) {
+            var query = nextQuery;
+            nextQuery = null;
+            search(query);
+        }
+    };
+
+
+    function search(query) {
+        currentQuery = query;
+        processing = true;
+        worker.postMessage({type: 'query', query: query}); 
+    }
+
+
+    // display
+
+    function displayResult(results) {    
+        var content = '';
+        for (r of results) {
+            content += '<a href="#" class="list-group-item list-group-item-action" class="suggestion-item">' + datalist[r.index] + '</a>'
```

Comment:
Instead of `'#'`, can you try `'/?search=<prof-name>'` ? Now, I have to click the button or hit enter to get the result.